Models/User.go

```diff
@@ -1,52 +1,63 @@
-package Models
+package model
 
-import(
+import (
+	"first-api/config"
 	"fmt"
-	"first-api/Config"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
 )
 
-// get all users Fetch all user data
- func GetAllUsers(user *[]User) (err error){
-	if err = Config.DB.Find(user).Error; err !=nil{
+type UserStore struct{}
+
+func (u *UserStore) CreateUser(user *User) error {
+	if err := config.DB.Create(user).Error; err != nil {
```

Comment:
add one more file in this package which will  be directly communicating with db layer and should expose the following method. 
   - Create(ctx,model interface{},queryBuilder) (interface{},error)
   - Update
   - Delete 
   - Find 
   - FindAll 
   
   
   This file should be responsible for db specific orchestration logic such as. 
       - converting db errors to application level error.